downloader.py

```diff
@@ -111,9 +121,49 @@ def grb2_to_array(filename):
     for i, level in enumerate(levels):
         assert(u[i]['level'] == level)
         assert(v[i]['level'] == level)
-        dataset[0][i] = u[i].data()[0][::2, ::2]
+        dataset[0][i] = u[i].data()[0][::2, ::2] # Takes the second element of every second array in data
         dataset[1][i] = v[i].data()[0][::2, ::2]
     return dataset
 
+## save data as npz file of ['data', 'timestamp (unix)', 'interval', 'levels']
+def combine_files():
+    file_list = os.listdir(f'{args.savedir}/temp')
+    filesets = [['']]
+    
+    for i in range(NUM_MEMBERS-1):
+        filesets.append([''])
+    for f in file_list:
+        n = int(f[26:28])
+        filesets[n-1].append(f)
```

Comment:
Please use glob.glob to improve readability for this function. If you use a globstring like `{args.savedir}/temp/{timestamp}_*_{n}.npy` you will get all of the member n wind files. If you sort these file names, they will be in chronological order.